### PR TITLE
Introduce 9c-rudolf service in all-in-one chart

### PR DESCRIPTION
**Pull request by moreal**
This pull request introduces a new optional service in `all-in-one` chart. You can see the service's source code at https://github.com/planetarium/9c-rudolf.

### CODE REVIEW

File: charts/all-in-one/templates/rudolf-service.yaml [hunk 1]
```diff
@@ -0,0 +1,67 @@
+{{ if .Values.rudolfService.enabled }}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  annotations:
+    eks.amazonaws.com/role-arn: {{ .Values.rudolfService.serviceAccount.roleArn }}
+  labels:
+    app.kubernetes.io/instance: {{ $.Chart.Name }}
+  name: {{ $.Chart.Name }}-9c-rudolf-iam-role
+  namespace: {{ $.Chart.Name }}
```
Comment (sky1045) [inline charts/all-in-one/templates/rudolf-service.yaml:10]: `$.Chart.Name` is always fixed to `9c-network`. so if you want to make this fields follow network name, you'd better use `$.Release.Name`

**Review by area363 — APPROVED** [review]

File: charts/all-in-one/templates/rudolf-service.yaml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        app: 9c-rudolf
+    spec:
+      containers:
+        - name: 9c-rudolf
+          env:
+            - name: DATABASE_URL
+              valueFrom: 
+                secretKeyRef:
+                  key: database-url
+                  name: 9c-rudolf-env
```
Comment (sky1045) [inline charts/all-in-one/templates/rudolf-service.yaml:42]: is it from external secret?

Comment (moreal) [reply charts/all-in-one/templates/rudolf-service.yaml:42]: Yes.

File: charts/all-in-one/templates/rudolf-service.yaml (hunk 1 above)
Comment (moreal) [reply charts/all-in-one/templates/rudolf-service.yaml:10]: Thanks. I'll update the all related parts. 🙏🏻

File: charts/all-in-one/templates/rudolf-service.yaml (hunk 2 above)
Comment (sky1045) [reply charts/all-in-one/templates/rudolf-service.yaml:42]: then you should also make externalSecret resource like below
https://github.com/planetarium/9c-infra/blob/main/charts/all-in-one/templates/secret-market-db.yaml

File: charts/all-in-one/templates/rudolf-service.yaml (hunk 1 above)
Comment (moreal) [reply charts/all-in-one/templates/rudolf-service.yaml]: I applied it in [c88b69e](https://github.com/planetarium/9c-infra/pull/787/commits/c88b69ec31fd2d16f1c226b846cbfb8426688b5c) commit.

File: charts/all-in-one/templates/rudolf-service.yaml (hunk 2 above)
Comment (moreal) [reply charts/all-in-one/templates/rudolf-service.yaml]: I applied it in [3987800](https://github.com/planetarium/9c-infra/pull/787/commits/3987800620e38ac3e8db4394374db08a6ea2934f) commit.

**Review by sky1045 — APPROVED** [review]
doesnt it use `Service` resource?

**Comment by moreal** [comment]
> doesnt it use `Service` resource?

@sky1045 Well.. It will be not opened for external endpoints, other pods. It will be accessed by only a specific lambda. But when I think about it again, Lambda ultimately needs an endpoint to access... so I think I need to figure out how to access the Pod directly. (not through Nginx)

**Review by sky1045 — APPROVED** [review]

**Pull request merged**